### ISSUE TITLE
OMP information pages fails on php > 7.4

### ISSUE THREAD

**Issue by OyvindLGjesdal** [pkp/pkp-lib#10016]
**Describe the bug**

When I try to visit about pages after updating the php version to 8 or higher, I get an error message:

```
 PHP Fatal error:  Declaration of InformationHandler::setupTemplate($request, $press) must be compatible with PKPHandler::setupTemplate($request) in /var/www/boap-omp/pages/information/InformationHandler.inc.php on line 102
```
When I downgrade to php 7.4 I see a similar message, but with a warning instead of as a fatal error.

I see this has been fixed in https://github.com/pkp/omp/blob/stable-3_4_0/pages/information/InformationHandler.php#L109

I don't know if updating in place, based on what is in stable-3.4.0, makes sense, something like:

```diff
+       function setupTemplate($request) {
+               parent::setupTemplate($request);
+                AppLocale::requireComponents(LOCALE_COMPONENT_APP_MANAGER); // FIXME needed?
+                if (!$request->getPress()->getSetting('restrictSiteAccess')) {
+                        $templateMgr = TemplateManager::getManager($request);
+                        $templateMgr->setCacheability(CACHEABILITY_PUBLIC);
+        }
+    }
 }
```
This seems to run without any errors, but does `!$request->getPress()->getSetting('restrictSiteAccess')` work, or is the if statement always public (I see the information pages)

**To Reproduce**
1. Go to a stock OMP with php 8 installed
2. Click on 'For the reader'
3. See error

**What application are you using?**
 OMP has been patched to 3.3.0.17, but I see the same snippet in stable 3.3 (https://github.com/pkp/omp/blob/stable-3_3_0/pages/information/InformationHandler.inc.php#L102).

**Comment by OyvindLGjesdal** [comment]
Saw in the git history that there was multiple changes for making PKP apps run on newer PHP in the LTS 3.3 version, so thought it could be worth posting. Not sure what the end php target for the LTS 3.3 branch is.

**Comment by asmecher** [comment]
@OyvindLGjesdal, what specific release of PHP8 are you using?

**Comment by OyvindLGjesdal** [comment]
```
php-fpm.x86_64                                                                           8.0.30-1.module+el8.10.0+1605+02e07af7                                                                         @appstream
```

from application stream in RHEL (on Rocky Linux 8)
https://access.redhat.com/support/policy/updates/rhel-app-streams-life-cycle#rhel8_full_life_application_streams

I noticed now when checking out the version that application stream just released 8.2, so I tested that also, with the same result.
